README.md

```diff
@@ -1,16 +1,16 @@
 [![Health IT Logo](https://healthit.com.au/wp-content/uploads/2019/06/HIT-proper-logo.png)](https://healthit.com.au)
 
-# pyConnectWise - An API library for ConnectWise Manage and ConnectWise Automate, written in Python
+# pyConnectWise - An API library for ConnectWise Manage and Labtech (ConnectWise Automate), written in Python
```

Comment:
I'm more inclined to reversing this and having it be ConnectWise Automate (Labtech) as having the current name (former name) makes more sense.